src/clj/game/core-rules.clj

```diff
@@ -65,30 +65,42 @@
     (let [drawn-this-turn (get-in @state [side :register :drawn-this-turn] 0)]
       (max (- max-draw drawn-this-turn) 0))))
 
+(defn draw-bonus
+  "Registers a bonus of n draws to the next draw (Daily Business Show)"
+  [state side n]
+  (swap! state update-in [:bonus :draw] (fnil #(+ % n) 0)))
+
 (defn draw
   "Draw n cards from :deck to :hand."
   ([state side] (draw state side 1 nil))
   ([state side n] (draw state side n nil))
   ([state side n {:keys [suppress-event] :as args}]
+   (swap! state assoc-in [side :register :most-recent-drawn] nil) ;clear the most recent draw in case draw prevented
```

Comment:
FYI (not necessary to implement), although there is no `dissoc-in` in Clojure, you can mimic it with `(swap! state assoc-in [side :register] dissoc :most-recent-drawn)`.